torch/jit/annotations.py

```diff
@@ -320,12 +321,13 @@ def try_ann_to_type(ann, loc):
             torch.jit._script._recursive_compile_class(ann, loc)
         return EnumType(_qualified_name(ann), get_enum_value_type(ann, loc), list(ann))
     if inspect.isclass(ann):
-        if hasattr(ann, "__torch_script_class__"):
-            return ClassType(_qualified_name(ann))
```

Comment:
grep `__torch_script_class__`: https://github.com/pytorch/pytorch/search?q=__torch_script_class__

There are a couple other places we have to update this

Reply:
Fixed